docs/reference/content/driver/getting-started/quick-start-pojo.md

```diff
@@ -0,0 +1,278 @@
+Person{id='591dbc2550852fa685b3ad18', name='Charles Babbage', age=45, address=Address{street='5 Devonshire Street', city='London', zip='W11'}}
+Person{id='591dbc2550852fa685b3ad19', name='Alan Turing', age=28, address=Address{street='Bletchley Hall', city='Bletchley Park', zip='MK12'}}
+Person{id='591dbc2550852fa685b3ad1a', name='Timothy Berners-Lee', age=61, address=Address{street='Colehill', city='Wimborne', zip='null'}}
+```
+
+## Specify a Query Filter
+
+To query for Person instance that match certain conditions, pass a filter object to the [`find()`]({{< apiref "com/mongodb/client/MongoCollection.html#find--">}}) method. 
+To facilitate creating filter objects, Java driver provides the [`Filters`]({{< apiref "com/mongodb/client/model/Filters.html">}}) helper.
+
+{{% note class="important" %}}
+When querying Pojos you *must* query against the document field name and not the Pojo's field name. 
+By default they are the same but it is possible to change how Pojo field names are mapped.
+{{% /note %}}
+
+
+### Get A Single Person That Matches a Filter
+
+For example, to find the first `Person` in the database that lives in `Wimborne` pass an [`eq`]({{<apiref  "com/mongodb/client/model/Filters.html#eq-java.lang.String-TItem-">}}) 
+filter object to specify the equality condition:
+
+```java
+somebody = collection.find(eq("address.city", "Wimborne")).first();
+System.out.println(somebody);
+```
+The example prints one document:
+
+```bash
+Person{id='591dbc2550852fa685b3ad1a', name='Timothy Berners-Lee', age=61, 
+       address=Address{street='Colehill', city='Wimborne', zip='null'}}
+```
+
+### Get All Person Instances That Match a Filter
+
+The following example returns and prints everyone where ``"age" > 30``:
+
+```java
+collection.find(gt("age", 30)).forEach(printBlock);
+```
+
+## Update Documents
+
+To update documents in a collection, you can use the collection's [`updateOne`]({{<apiref "com/mongodb/client/MongoCollection.html#updateOne-org.bson.conversions.Bson-org.bson.conversions.Bson-">}})  and  [`updateMany`]({{<apiref "com/mongodb/async/client/MongoCollection.html#updateMany-org.bson.conversions.Bson-org.bson.conversions.Bson-">}}) methods.
+
+Pass to the methods:
+
+- A filter object to determine the document or documents to update. To facilitate creating filter objects, Java driver provides the [`Filters`]({{< apiref "com/mongodb/client/model/Filters.html">}}) helper. To specify an empty filter (i.e. match all Persons in a collection), use an empty [`Document`]({{< apiref "org/bson/Document.html" >}}) object.
+
+- An update document that specifies the modifications. For a list of the available operators, see [update operators]({{<docsref "reference/operator/update-field">}}).
+
+The update methods return an [`UpdateResult`]({{<apiref "com/mongodb/client/result/UpdateResult.html">}}) which provides information about the operation including the number of documents modified by the update.
+
+### Update a Single Person
```

Comment:
Add an example that uses replaceOne as well, showing that you can pass a Person instance directly to the method.